content/en/blog/2023/dlb-connection-balancing/index.md

```diff
@@ -0,0 +1,307 @@
+
+Now, Envoy has an integrated Intel® Dynamic Load Balancing (Intel®DLB) connection load balancing to accelerate in big connection cases like ingress gateway.
+
+## How Intel® Dynamic Load Balancing accelerates connection load balancing in Envoy
+
+Intel DLB is a hardware managed system of queues and arbiters connecting producers and consumers. It is a PCI device envisaged to live in the server CPU [uncore](https://en.wikipedia.org/wiki/Uncore) and can interact with software running on cores, and potentially with other devices.
+
+Intel DLB implements the following load balancing features:
+
+- Offloads queue management from software — useful where there are significant queuing-based costs
+    - Especially with multi-producer / multi-consumer scenarios and enqueue batching to multiple destinations
+    - The overhead locks are required to access shared queues in the software. Intel DLB implements lock-free access to shared queues
+- Dynamic, flow aware load balancing and reordering
+    - Ensures equal distribution of tasks and better CPU core utilization. Can provide flow-based atomicity if required
+    - Distributes high bandwidth flows across many cores without loss of packet order
+    - Better determinism and avoids excessive queuing latencies
+    - Uses less IO memory footprint and saves DDR Bandwidth
+- Priority queuing (up to 8 levels) — allows for QOS
+    - Lower latency for traffic that is latency sensitive
+    - Optional delay measurements in the packets
+- Scalability
+    - Allows dynamic sizing of applications, seamless scale up/down
+    - Power aware; application can drop workers to lower power state in cases of lighter load
+
+There are three types of load balancing queues:
+
+- Unordered: For multiple producers and consumers. The order of tasks is not important, and each task is assigned to the processor core with the lowest current load.
+- Ordered: For multiple producers and consumers where the order of tasks is important. When multiple tasks are processed by multiple processor cores, they must be rearranged in the original order.
+- Atomic: For multiple producers and consumers, where tasks are grouped according to certain rules. These tasks are processed using the same set of resources and the order of tasks within the same group is important.
+
+An ingress gateway is expected to process as much data as possible as quickly as possible, so Intel DLB connection load balancing uses an unordered queue.
+
+## How to use Intel DLB connection load balancing in Istio
+
+With the 1.17 release, Istio officially supports Intel DLB connection load balancing.
+
+The following steps show how to use Intel DLB connection load balancing in an Istio [Ingress Gateway](/docs/tasks/traffic-management/ingress/ingress-control/) in an SPR (Sapphire Rapids) machine and Kubernetes is ready.
```

Comment:
and Kubernetes is ready? I don't understand this last part.

Reply:
Istio requires a K8s cluster.